Source/Core/DolphinQt/Config/NewPatchDialog.cpp

```diff
@@ -164,6 +179,23 @@ QGroupBox* NewPatchDialog::CreateEntry(PatchEngine::PatchEntry& entry)
             value->setPalette(palette);
           });
 
+  connect(comparand, qOverload<const QString&>(&QLineEdit::textEdited),
```

Comment:
I missed this at first, but can you deduplicate the signal handler? It's the same code as lines 148-180, just with different widget variables...